core/async/README.md

```diff
@@ -0,0 +1,8 @@
+# Core Async Helpers
+
+This crate contains helpers related to common asynchronous programming patterns
+used in nearcore:
+
+* messaging: common interfaces for sending messages between components.
+* test_loop: a event-loop-based test framework that can test multiple components
+  together in a synchronous way.
```

Comment:
drive-by comment:

What's the plan for this crate? do we want to replace actix ? run this in parallel to actix?

Reply:
It's an abstraction that makes our things not have to care about whether actix is being used under the hood.

messaging::Sender is very much similar to MsgRecipient in near-network, and is intended to be a replacement. Problem right now is that MsgRecipient has an async api as well. That is not yet resolved. Regarding actix, we can continue to use actix without any problems, but as we migrate all actors to use this abstraction (which is quite simple to do), we are also free to move away from actix.